kibana-reports/server/utils/constants.ts

```diff
@@ -13,4 +13,4 @@
  * permissions and limitations under the License.
  */
 
-export const POLL_INTERVAL = 1000 * 60 * 5; // in milliseconds (5 min)
+export const POLL_INTERVAL = 1000 * 60 * 0.5; // in milliseconds (5 min)
```

Comment:
is the 5 min still accurate? even after changing from `5` to `0.5`?

Reply:
good point, changed it back